cypress/e2e/shared/dashboardsIndex.test.ts

```diff
@@ -718,4 +719,157 @@ describe('Dashboards', () => {
     cy.getByTestID('button-copy').click()
     cy.getByTestID('notification-success--children').should('be.visible')
   })
+  it('changes time range', () => {
+    const dashName = 'dashboard'
+    const newDate = new Date()
+    const now = newDate.toISOString()
+    const pastDate = new Date(new Date() - 1000 * 60 * 60 * 2)
+    const past = pastDate.toISOString()
+    cy.get<Organization>('@org').then(({id}: Organization) =>
+      cy.createDashboard(id, dashName).then(({body}) => {
+        cy.createCell(body.id).then(cell1Resp =>
+          cy.createView(body.id, cell1Resp.body.id).then(() =>
+            cy.fixture('routes').then(({orgs}) =>
+              cy.get<Organization>('@org').then(({id}: Organization) => {
+                cy.visit(`${orgs}/${id}/dashboards-list`)
+                cy.getByTestID('tree-nav').then(() =>
+                  cy
+                    .intercept('POST', '/api/v2/query?*')
+                    .as('loadQuery')
+                    .then(() =>
+                      cy.writeLPData({
+                        lines: genCurve({points: 3500, freq: 6, shift: 13}),
+                        offset: `${8 * 24 * 60}m`,
+                        stagger: `${(8 * 24 * 60) / 3499}m`,
+                      })
+                    )
+                )
+              })
+            )
+          )
+        )
+      })
+    )
+    cy.getByTestID('dashboard-card--name').click()
+    cy.getByTestID('cell-context--toggle').click()
+    cy.getByTestID('cell-context--configure').click()
+    cy.getByTestID('selector-list curve').click()
+    cy.getByTestID('selector-list p').click()
```

Comment:
There is a lot of clicking here with no assertions of visibility. This will cause the test to be flaky. I'd recommend adding some assertions of visibility between the clicks

Reply:
I pushed another commit which adds assertion that ensures visibility of new window's page header and another assertion that checks giraffe layer visibility. Thanks for the recommendation